src/main/java/frc/robot/Manipulator.java

```diff
@@ -307,57 +388,21 @@ public boolean isFinished() {
 		public void end(boolean isfinished) {
 			this.manipulator.arm.setWinchVoltage(0);
 			this.manipulator.grabber.setGrabberVoltage(0);
-			this.manipulator.grabber.setWristPercent(0);
+			this.manipulator.grabber.setWristPercent(Grabber.WRIST_NEUTRAL_PERCENT);
 		}
 
 		@Override
 		public void initSendable(SendableBuilder b) {
-			b.addDoubleProperty("Winch Voltage Setpoint", ()->this.arm_percent.getAsDouble()*ARM_WINCH_VOLTAGE_SCALE, null);
-			b.addDoubleProperty("Grabber Voltage Setpoint", ()->this.grab_percent.getAsDouble()*GRAB_CLAW_VOLTAGE_SCALE, null);
-			b.addDoubleProperty("Wrist Position Setpoint", ()->this.wrist_set.getAsDouble(), null);
+			b.addDoubleProperty("Winch Voltage Setpoint",
+				()->this.is_arm_locked ? ARM_WINCH_LOCK_VOLTAGE : (this.arm_rate.getAsDouble() * ARM_WINCH_VOLTAGE_SCALE), null);
```

Comment:
Are the "lock voltages" something that was determined experimentally?  Or is this just a placeholder for now?  Wouldn't it depend on load or arm position?

Reply:
They are temporary - to be determined tonight.